test/data/check-uk-visa-responses-and-expected-results.yml

```diff
@@ -630,7 +630,7 @@
   - kuwait
   - work
   - six_months_or_less
-  :next_node: :outcome_work_m
+  :next_node: :outcome_work_waiver
```

Comment:
It looks as if this change was the result of this commit: 8053b80eb930b8bab176d2af43b148a4656fdda0 which was part of #2524. I'm therefore happy to include it.